network-api/networkapi/utility/middleware.py

```diff
@@ -1,30 +1,30 @@
-from django.http.response import HttpResponseRedirectBase

```

Comment:
I think this is because I did a global CR+LF => LF line ending replacement... were CR+LF crucial here?